libs/gi/db/src/Database/ArtCharDatabase.ts

```diff
@@ -64,6 +66,7 @@ export class ArtCharDatabase extends Database {
 
     // Depends on arts
     this.optConfigs = new OptConfigDataManager(this)
+    this.generatedBuildList = new GeneratedBuildListDataManager(this)
```

Comment:
You need to initialize `generatedBuildList` before `optConfigs`, because in the validation of optConfig, it validates the id of the list, 
```suggestion
    // Depends on arts
    this.generatedBuildList = new GeneratedBuildListDataManager(this)
    // Depends on arts and generatedBuildList
    this.optConfigs = new OptConfigDataManager(this)
```

Reply:
Fixed